distribution/lib/Standard/Base/0.0.0-dev/src/Network/HTTP/Response.enso

```
@@ -52,6 +66,14 @@ type Response
         content_type_optional = self.internal_http_response.headers.firstValue "Content-Type"
         if content_type_optional.isPresent then content_type_optional.get else Nothing
 
+    ## Get the response content length in bytes.
+    content_length : Integer | Nothing
+    content_length self =
+        content_length_optional = self.internal_http_response.headers.firstValue "Content-Length"
+        if content_length_optional.isPresent.not then Nothing else
+            length_text = content_length_optional.get
+            Integer.parse length_text
```

Shall we maybe raise a dataflow error if content-length is not defined?